frontend/src/app/features/book/components/wanted-books/wanted-books.component.html

```
@@ -0,0 +1,195 @@
+      </tr>
+
+      @if (expandedRows[book.id]) {
+        <tr>
+          <td colspan="7" class="bg-surface-ground p-3">
+            @if (!jobHistory[book.id]) {
+              <div class="text-muted-color text-sm">Loading history...</div>
+            }
+            @if (jobHistory[book.id] && jobHistory[book.id].length === 0) {
+              <div class="text-muted-color text-sm">No search history yet.</div>
+            }
+            @if (jobHistory[book.id] && jobHistory[book.id].length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)